proto/iotics/api/feed.proto

```diff
@@ -36,6 +36,9 @@ service FeedAPI {
   // Shares a new sample of data for the given feed which any (interest) subscribers can receive.
   rpc ShareFeedData(ShareFeedDataRequest) returns (ShareFeedDataResponse) {}
 
+  // Shares feed data over a stream, any sharing error will return the error and close the stream
+  rpc StreamFeedData(stream ShareFeedDataRequest) returns (ShareFeedDataResponse) {}
```

Comment:
This should indicate that currently only streamed sharing of a **single** feed is supported

Reply:
Note: cannot share to different feeds over the same stream. Will be added.